runtime/vm/swalk.c

```diff
@@ -346,8 +346,9 @@ UDATA  walkStackFrames(J9VMThread *currentThread, J9StackWalkState *walkState)
 		if (walkFrame(walkState) != J9_STACKWALK_KEEP_ITERATING) {
 			goto terminationPoint;
 		}
-		walkState->previousFrameFlags = walkState->frameFlags;
 resumeInterpreterWalk:
+		walkState->previousFrameFlags = walkState->frameFlags;
+		walkState->resolveFrameFlags = 0;
```

Comment:
Is the intention that the `resolveFrameFlags` are only set if the current frame is a resolveFrame?  And otherwise cleared?

Overall, I have to wonder why we bother with a separate resolveFrameFlags in that case.

Reply:
Agreed - a lot of this is historical cruft, but fixing that is for a different day.

resolveFrameFlags is set when the previously-walked frame is a resolve frame.  Generally only applicable if the current frame is a JIT frame, but this seemed the safest and most "correct" way to fix it, since there clearly is one case where the resolve frame is not on top of a JIT frame.

Reply:
Thanks for confirming that I understood the fix :)